crates/typst/src/model/heading.rs

```diff
@@ -65,18 +65,18 @@ pub struct HeadingElem {
     /// ```
     pub level: Smart<NonZeroUsize>,
 
-    /// The relative nesting depth of the heading, starting from one. This is
+    /// The relative nesting depth of the heading, starting from `{1}`. This is
     /// combined with `{offset}` to compute the actual `{level}`.
     ///
     /// This is set by the heading syntax, such that `[== Heading]` creates a
-    /// heading with logical depth 2, but actual level `{offset + 2}`. If you
-    /// construct a heading manually, you should typically prefer this over
-    /// setting the absolute `level`.
+    /// heading with logical `{depth}` of `{2}`, but the actual `{level}` is
+    /// `{offset + 2}`. If you construct a heading manually, you should
```

Comment:
`offset + 2` is the important highlight. Previously, it stood out, now it drowns in a sea of other highlights. It is true that the other words you emphasized are also properties, but this is not about using them but rather about understanding the concepts behind it. The change of typeface should always be considered as it disrupts the reading flow.

Reply:
I don't understand what you mean by that. Do you want me to completely revert the changes and only leave the "behavior" fix? Or do some other changes also can stay?

Reply:
I also wanted to added syntax highlighting here:
![image](https://github.com/typst/typst/assets/37143421/8ab90d1a-44ed-4701-87ec-6556f74a8502)

since it already has backticks in place. Will those changes be ok to do?

Reply:
Behavior fix is okay and "logical depth of 2" is also an improvement.

Since the `shift` changes are unrelated, let's do them after the release.